admin-ui/app/project/create/index.tsx

```diff
@@ -3,10 +3,10 @@ import { Button, Grid, Paper, TextField, Typography } from "@mui/material";
 import Checkbox from "@mui/material/Checkbox";
 import Link from "next/link";
 import { useRouter } from "next/navigation";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import { applicationType } from "../../../component/common/data/project/application.type";
```

Comment:
why do you need this data

Reply:
We need this data for different types of Application based on business like 
B2C
B2B
etc..